examples/synthetics_mesh_example_test.go

```diff
@@ -284,3 +285,26 @@ func (m metricsMatrixGRPC) getMetricGRPC(fromAgent string, toAgent string) (*syn
 
 	return metric, true
 }
+
+func pickTestID() (string, error) {
+	client, err := NewClient()
+	if err != nil {
+		return "", err
+	}
+	ctx, cancel := context.WithTimeout(context.Background(), 300*time.Second)
+	defer cancel()
+
+	getAllResp, err := client.SyntheticsAdmin.ListTests(ctx, &syntheticspb.ListTestsRequest{})
+	if err != nil {
+		return "", err
+	}
+
+	if getAllResp.Tests != nil {
+		for _, test := range getAllResp.GetTests() {
+			if test.GetType() == "application_mesh" {
+				return test.GetId(), nil
+			}
+		}
+	}
+	return "", fmt.Errorf("No tests with type application_mesh: %v", err)
```

Comment:
I would include information that there is no such tests but specifically for requested Kentik Account 
err here is most likely always nil?